src/term.js

```diff
@@ -732,75 +732,83 @@ Terminal.prototype.open = function(parent) {
   // Draw the screen.
   this.refresh(0, this.rows - 1);
 
-  // Initialize global actions that
-  // need to be taken on the document.
-  this.initGlobal();
+  if (this.options.noEvents) {
```

Comment:
Yes, this must've slipped by -- and good point regarding the positive naming, Il'l make these changes.